com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/PollServiceDeleteOperationsExecution.java

```diff
@@ -26,16 +29,14 @@ public PollServiceDeleteOperationsExecution(EventsGetter eventsGetter) {
     @Override
     protected List<CloudServiceExtended> computeServicesToPoll(ExecutionWrapper execution,
         Map<String, ServiceOperationType> triggeredServiceOperations) {
-        List<String> servicesToDeleteNames = StepsUtil.getServicesToDelete(execution.getContext());
-        List<CloudServiceExtended> servicesToDelete = getServicesData(servicesToDeleteNames, execution);
-        return getServicesWithTriggeredOperations(servicesToDelete, triggeredServiceOperations);
+        Map<String, CloudServiceExtended> servicesToDeleteData = getServicesData(execution);
+        List<CloudServiceExtended> services = new ArrayList<>();
```

Comment:
Just modify `getServicesWithTriggeredOperations` to accept a `Collection` object instead of `List`.